src/docs/Contributing/PatternEnhancement/index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Pattern enhancement'
+order: 4
+---
+
+When using MOZAIC patterns, it may happen that they do not fully meet your needs. For example:
+
+- You want to use another type of pattern variation (color, size, etc...)
+- You want to use the pattern in an other usage than the one defined by default in MOZAIC
+- You want to make the pattern responsive to a resolution not implemented by default in MOZAIC
+- You want to make the pattern more accessible
+- etc...
+
+In these cases where an improvement of the pattern seems necessary, two possibilities are available to you:
+
+1. Suggest a pattern enhancement
+2. Submit a pattern enhancement
+
+## Suggest a pattern enhancement
+
+### Talk to the MOZAIC core team about it
+
+One way to suggest a pattern enhancement is to discuss it with the MOZAIC core team.
+
+For this purpose, the dedicated **#mozaic-support** channel is available on [Adeo's global Slack account](https://adeo-tech-community.slack.com/).
+
+
+### Create an Issue on Github
+
+Another method _(recommended)_ is to create an **Issue** in the [MOZAIC project on Github](https://github.com/adeo/mozaic-design-system/issues/new/choose) using the dedicated template.
+
+<br/>
+
+***
+
+## Submit a pattern enhancement

```

Comment:
you can says that pattern enhancement require only core team validation, and to me this seems good

Reply:
It's ok. I added this to the file.